discovery-infra/tests/config/global_configs.py

```diff
@@ -65,10 +65,10 @@ def _get_iso_download_path(infra_env_name: str):
 
     def __post_init__(self):
         super().__post_init__()
-        if self.infra_env_name is None or isinstance(self.infra_env_name, str):
-            self.infra_env_name = InfraEnvName()
+        if self.entity_name is None or isinstance(self.entity_name, str):
+            self.entity_name = InfraEnvName()
```

Comment:
why would it be an infra env and not a cluster name?

Reply:
WDYM?
Those are two separate objects

Reply:
How should I infer that the default "entity name" stands for the infra env? Why do we have even this distinction between a cluster name and a general entity name?

Reply:
ok I missed that it happens under `InfraEnvConfig`.